SparkPost/matchbox
# UX-583 add LinkedInSimple icon

### What Changed

- Added new LinkedInSimple icon to matchbox-icons

### How To Test or Verify

- `npm run start:libby` in matchbox root
- verify LinkedInSimple icon exists and looks right in libby


### PR Checklist
